metadata.py

```diff
@@ -0,0 +1,67 @@
+"""
+The purpose of this file is to define the metadata of the app with minimal imports. 
+
+DO NOT CHANGE the name of the file
+"""
+
+from mmif import DocumentTypes, AnnotationTypes
+
+from clams.appmetadata import AppMetadata
+
+
+# DO NOT CHANGE the function name 
+def appmetadata() -> AppMetadata:
+    """
+    Function to set app-metadata values and return it as an ``AppMetadata`` obj.
+    Read these documentations before changing the code below
+    - https://sdk.clams.ai/appmetadata.html metadata specification. 
+    - https://sdk.clams.ai/autodoc/clams.appmetadata.html python API
+    
+    :return: AppMetadata object holding all necessary information.
+    """
+    
+    # Basic Information
+    metadata = AppMetadata(
+        name="Chyron Recognition",
+        description="This tool detects chyrons and generates time segments.",
+        app_license="MIT",  
+        identifier="chyron-recognition",
+        url="https://github.com/clamsproject/app-chyronrecognition",
+    )
+
+    # I/O Spec
+    metadata.add_input(DocumentTypes.VideoDocument)
+
+    metadata.add_output(AnnotationTypes.TimeFrame, properties={"frameType":"string"})
```

Comment:
`frameTyep` = `chyron` maybe?

Reply:
Or [`lower-third`](https://en.m.wikipedia.org/wiki/Lower_third)? Chyron sounds more specific, perhaps too much specific.

Reply:
Chyron does sound more specific but maybe it isn't. It is defined as lower part or bottom part of the screen, which would be less specific than lower-third.